modules/ROOT/pages/upgrade-self-managed.adoc

```diff
@@ -0,0 +1,90 @@
+= Upgrade Runtime Fabric for Self-Managed Kubernetes
+
+To upgrade Anypoint Runtime Fabric for Self-Managed Kubernetes, upgrade Runtime Fabric components and the `rtfctl` command-line utility.
+
+== About Component Upgrades
+
+Upgrading Runtime Fabric components means upgrading services that:
```

Comment:
You need a line between the intro sentence and the bullet items or the list doesn't render correctly in the output.

Reply:
```suggestion
Upgrading Runtime Fabric components means upgrading services that:

```